numpy/core/tests/test_umath.py

```diff
@@ -17,6 +18,15 @@
     _gen_alignment_data, assert_array_almost_equal_nulp, assert_warns
     )
 
+def get_glibc_version():
+    ver = 0.0
+    try:
+        ver = float(os.confstr('CS_GNU_LIBC_VERSION').rsplit(' ')[1])
+    except:
+        print("Couldn't capture GLIBC version")
```

Comment:
Might a warning not be more suitable here compared to just `print`?

Reply:
The try-except block is mainly so not to crash on Windows OS where the `os` module doesn't have `confstr` module. I could use `sys.platform` to verify that it is linux too.